micro-deps/micro-deps/src/main/groovy/com/ofg/infrastructure/discovery/ServiceConfigurationResolver.groovy

```diff
@@ -88,4 +90,12 @@ class ServiceConfigurationResolver {
         return parsedConfiguration[dependencyName]
     }
 
-} 
+    String getLoadBalancerTypeOf(String dependencyPath) {
```

Comment:
This should take strongly typed `ServicePath` as an argument. That's why it was invented, to avoid ambiguity.